Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue11829.cs

```diff
@@ -0,0 +1,114 @@
+﻿using Xamarin.Forms.Internals;
+using Xamarin.Forms.CustomAttributes;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+using Xamarin.Forms.Core.UITests;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
```

Comment:
This should be automated with a platform test on iOS - we can verify that the Control's AttributedText has the strikethrough attribute after the Label has LineHeight and TextDecoration set.